codewiki/cli/commands/generate.py

```
@@ -172,61 +185,68 @@ def generate_command(
+
         # Run generation
         job = generator.generate()
-        
+
         # Post-generation
         generation_time = time.time() - start_time
-        
+
         # Get repository info
         repo_url = None
         commit_hash = get_git_commit_hash(repo_path)
```

Ruff (0.14.10)

Local variable `commit_hash` is assigned to but never used

Remove assignment to unused variable `commit_hash`

(F841)